src/components/Appbar/index.tsx

```diff
@@ -58,14 +58,16 @@ const Appbar = ({ setPrefersDarkMode }: AppbarProps) => {
   const queryClient = useQueryClient();
   const dispatch = useDispatch<AppDispatch>();
 
-  const handleClick = () => history.push(Routes.ROOT);
-
   const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
   const [isSidebarOpen, setSidebarOpen] = useState(false);
 
   const isUserLogged = useMemo(() => !!user, [user]);
   const prefersDarkMode = useMemo(() => theme.palette.mode === "dark", [theme]);
 
+  const handleClick = () => {
+    return isUserLogged === true ? history.push(Routes.PRODUCT): history.push(Routes.ROOT)
```

Comment:
What do you think of creating a variable to store `targetRoute` to improve code semantic and avoid having `history.push` 2 times?
```suggestion
    const targetRoute = isUserLogged === true ? Routes.PRODUCT : Routes.ROOT;
    
    history.push(targetRoute):
```

Reply:
I think those are great suggestions! I realize now why I don't need the return statement and '=== true' comparison. Thanks